libs/shared/src/features/audio-playback/audio-playback.service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@angular/core';
+import { Router, NavigationStart } from '@angular/router';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AudioService {
+  private audio: HTMLAudioElement;
```

Comment:
+1 - nice you were able to do get things working with the standard html audio api. In another project I've used [HowlerJS](https://howlerjs.com/), however it looks like the api should be sufficient for now